src/SAWScript/Crucible/MIR/Builtins.hs

```diff
@@ -916,173 +932,6 @@ verifySimulate opts cc pfs mspec args assumes top_loc lemmas globals _checkSat m
 -- Utilities
 --------------------------------------------------------------------------------
 
--- | Check if two 'Mir.Ty's are compatible in SAW. This is a slightly coarser
--- notion of equality to reflect the fact that MIR's type system is richer than
--- Cryptol's type system, and some types which would be distinct in MIR are in
```

Comment:
This code simply moved to `SAWScript.Crucible.MIR.ResolveSetupValue`—no other changes were applied.